rpm/generic/zfs.spec.in

```diff
@@ -290,6 +291,7 @@ exit 0
 %config(noreplace) %{_initconfdir}/zfs
 %endif
 %config(noreplace) %{_sysconfdir}/%{name}
+%config(noreplace) %{_sysconfdir}/sudoers.d/*
```

Comment:
This sudoer's file needs to have the permissions `440` and be owned by root.  Add the keyword `%attr(440, root, root)`.